Shared/Samples/Edit feature attachments/EditFeatureAttachmentsView.swift

```diff
@@ -0,0 +1,228 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct EditFeatureAttachmentsView: View {
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    /// The location that the user tapped on the map.
+    @State private var screenPoint: CGPoint?
+    /// A Boolean value indicating whether the attachment sheet is showing.
+    @State private var attachmentSheetIsPresented = false
+    
+    var body: some View {
+        MapViewReader { mapProxy in
+            MapView(map: model.map)
+                .callout(
+                    placement: $model.calloutPlacement.animation(
+                        model.calloutShouldOffset ? nil : .default.speed(2))
+                ) { _ in
+                    HStack {
+                        CalloutView(model: model)
+                            .padding(6)
+                        Button {
+                            attachmentSheetIsPresented = true
+                        } label: {
+                            Image(systemName: "exclamationmark.circle")
+                        }
+                        .sheet(isPresented: $attachmentSheetIsPresented) {
+                            NavigationView {
+                                AttachmentSheetView(model: model)
+                            }
+                        }
+                        .padding(8)
+                    }
+                }
+                .onSingleTapGesture { tap, _ in
+                    self.screenPoint = tap
```

Comment:
```suggestion
                .onSingleTapGesture { screenPoint, _ in
                    self.screenPoint = screenPoint
```
`screenPoint` should be used here for clarity. This will not cause a naming conflict since `self` is used.